cypress/integration/buy-shirt.spec.ts

```diff
@@ -1,25 +1,38 @@
-describe("Buy a t-shirt", () => {
+import { MenuContentPage } from "../page/index"
+import { ShoppingCartPage } from "../page/index"
+import { ShippingCartPage } from "../page/index"
+import { LoginPage } from "../page/index"
+import { PaymentStep } from "../page/index"
+import { AdressStep } from "../page/index"
+import { ProductsListPage } from "../page/index"
+
+const menuContentPage = new MenuContentPage()
+const shoppingCartPage = new ShoppingCartPage()
+const shippingCartStep = new ShippingCartPage()
+const loginPage = new LoginPage()
+const paymentStep = new PaymentStep()
+const adressStep = new AdressStep()
+const productsListPage = new ProductsListPage()
```

Comment:
please move the initialization of the page objects to a before function, we do a similar initialization on `cypress/integration/dresses-list.spec.ts` file

Reply:
Done